modules/python/test/test_imgproc.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
```

Comment:
`imgproc/misc/python/test` is preferable.

Example: https://github.com/opencv/opencv/tree/4.10.0/modules/ml/misc/python/test

Reply:
Done